src/Meilisearch/MeilisearchClient.cs

```diff
@@ -314,6 +315,22 @@ await _http.PostAsJsonAsync("keys", keyOptions, Constants.JsonSerializerOptionsW
             return await responseMessage.Content.ReadFromJsonAsync<Key>(cancellationToken: cancellationToken).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Updates an API key for the Meilisearch server.
+        /// </summary>
+        /// <param name="keyUid">Unique identifier of the API key.</param>
+        /// <param name="keyOptions">The options of the API key.</param>
+        /// <param name="cancellationToken">The cancellation token for this call.</param>
+        /// <returns>Returns the created API key.</returns>
+        public async Task<Key> UpdateKeyAsync(string keyUid, Key keyOptions, CancellationToken cancellationToken = default)
+        {
+            var json = JsonSerializer.Serialize(keyOptions, Constants.JsonSerializerOptionsRemoveNulls);
+            var request = new HttpRequestMessage(new HttpMethod("PATCH"), $"keys/{keyUid}");
```

Comment:
Is this a good suggestion in this case? I like this way to instantiate objects 🏒 

```c#
var request = new HttpRequestMessage(new HttpMethod("PATCH"), $"keys/{keyUid}") {
       Content = new StringContent(json, Encoding.UTF8, "application/json")
};
```

Reply:
This is fine for me